docs/mast/mast_mastquery.rst

```diff
@@ -3,26 +3,118 @@
+                         ToO Comet  NIRSPEC  ZTF (C/2022 E3)
+                         ToO Comet  NIRSPEC  ZTF (C/2022 E3)
+                         ToO Comet  NIRSPEC  ZTF (C/2022 E3)
+   De-Mystifying SPRITEs with JWST  NIRSPEC      SPIRITS18nu
+                         ToO Comet  NIRSPEC  ZTF (C/2022 E3)
+                               ...      ...              ...
+                         ToO Comet  NIRSPEC  ZTF (C/2022 E3)
+                         ToO Comet  NIRSPEC  ZTF (C/2022 E3)
+                         ToO Comet  NIRSPEC  ZTF (C/2022 E3)
+   Length = 319 rows
+
+
+TESS Queries
+------------
+
+TESS queries have 2 types of filtered services. To output a table and specify
+columns for a TESS query, use TIC or CTL services with '.Rows' on the end
+(e.g. `Mast.Catalogs.Filtered.Tic.Rows
+<https://mast.stsci.edu/api/v0/_services.html#MastCatalogsFilteredTicRows>`__).
+
+.. doctest-remote-data::
+
+   >>> from astroquery.mast import Mast
+   ...
+   >>> observations = Mast.mast_query('Mast.Catalogs.Filtered.Tic.Rows',
+   ...                                columns = 'id',
+   ...                                dec = [{'min': -90, 'max': -30}],
+   ...                                Teff = [{'min': 4250, 'max': 4500}], 
+   ...                                logg = [{'min': 4.5, 'max': 5.0}],
+   ...                                Tmag = [{'min': 8, 'max': 10}])
+   >>> print(observations) # doctest: +IGNORE_OUTPUT
+      ID
+   ---------
+   320274328
+   408290683
+   186485314
+   395586623
+   82007673
+   299550797
+   ...
+   333372236
+   394008846
+   261525246
+   240766734
+   240849919
+   219338557
+   92131304
+   Length = 814 rows
 
-The basic MAST query function returns query results as an `~astropy.table.Table`.
+TESS services without '.Rows' in the title are used for count queries and will
+not mask the output tables using the columns parameter. Additionally, using a
+'.Rows' service for a count query will result in an error.
 
 .. doctest-remote-data::
 
    >>> from astroquery.mast import Mast
    ...
-   >>> service = 'Mast.Caom.Cone'
-   >>> params = {'ra':184.3,
-   ...           'dec':54.5,
-   ...           'radius':0.2}
-   >>> observations = Mast.service_request(service, params)
+   >>> observations = Mast.mast_query('Mast.Catalogs.Filtered.Tic.Rows',
+   ...                                columns = 'COUNT_BIG(*)',
+   ...                                dec = [{'min': -90, 'max': -30}],
+   ...                                Teff = [{'min': 4250, 'max': 4500}], 
+   ...                                logg = [{'min': 4.5, 'max': 5.0}],
+   ...                                Tmag = [{'min': 8, 'max': 10}])
+   Traceback (most recent call last):
+   ...
+   astroquery.exceptions.RemoteServiceError: Incorrect syntax near '*'.
+
+
+Cone Searches
+=============
+
+MAST's cone search services use the parameters 'ra', 'dec', and 'radius' and return
+a table of observations with all columns present.
```

Comment:
Is this the service that ignores when the user specify which `columns` to return? If so, be sure to mention this somewhere in here before the example. Is there error handling to catch when this happens?

Reply:
Nvm! I see that you describe this a few examples below. You can ignore this comment